antarest/study/service.py

```diff
@@ -206,6 +224,54 @@ def assert_raw(study: Study) -> RawStudy:
     return study
 
 
+class StudySettingsDTO(AntaresBaseModel):
+    time_series: TimeSeriesConfiguration
+    general: GeneralConfig
+    advanced_parameters: AdvancedParameters
+    adequacy_patch: AdequacyPatchParameters
+    optimization: OptimizationPreferences
+    thematic_trimming: ThematicTrimming
+    playlist: Playlist
+
+
+class StudyXpansionDTO(AntaresBaseModel):
+    settings: XpansionSettings
+    candidates: list[XpansionCandidate]
+    constraint: bytes
+
+
+class StudyShortTermStorageDTO(AntaresBaseModel):
+    cluster: STStorage
+    constraints: list[STStorageAdditionalConstraint]
+
+
+class StudyAreasDTO(AntaresBaseModel):
+    id: str
+    properties: AreaProperties
+    ui: dict[str, Any]
+    thermals: list[ThermalCluster]
+    renewables: list[RenewableCluster]
+    st_storages: list[StudyShortTermStorageDTO]
+    hydro: HydroProperties
+
+
+class StudyDataDTO(AntaresBaseModel):
+    """
+    DTO representing data of the whole study.
+    """
+
+    name: str
+    version: StudyVersionStr
+    path: str | None
```

Comment:
Seems weird to me to have this here, it's not actual study data

Reply:
hmmm ok then, there's already some "metadata" here anyway. Indeed we could rename it to folder, path seems a little more ambiguous (could mean the full path of the study or something like that)